website/src/components/Schema/index.js

```diff
@@ -85,11 +87,17 @@ const Schema = ({ schema, menuRef }) => {
     },
     {
       Header: 'Data Tier',
-      Cell: ({ original: { meta } }) => {
-        if (!meta) return null;
-        const { primaryId, core } = meta;
-        const type = primaryId ? TAG_TYPES.id : core ? TAG_TYPES.core : TAG_TYPES.extended;
-        return <Tag type={type} />;
+      Cell: ({ original }) => {
+        const meta = get(original, 'meta', {});
+        const required = get(original, 'restrictions.required', false);
+
+        if (isEmpty(meta) && !required) {
+          return <Tag type={TAG_TYPES.extended} />;
```

Comment:
what if no meta AND required?
This logic allows a field to have no tags which I think should be impossible.

Reply:
ok it will always have a tag... but this allows required fields that aren't marked CORE to be listed as EXTENDED

Reply:
@rosibaj @joneubank im struggling to understand which of these are exclusive.
is this logic correct:
 if(meta.core === false && meta.primaryId === false && restrictions.required === false) || meta is empty {show extended}

Reply:
There are 3 states for the Data Tier Column: Id, Core, Extended
- If primaryid = true, then id
- If core = true, then true 
- If neither of these, then extended.  

If primary then required MUST be true, if Core Required MIGHT be true, If Extended its (technically) possible for it to be required but in practice none should be.  

Not sure why required matters here?

Reply:
@joneubank updated :)

Reply:
thanks!